hmpps-prisoner-search/src/test/kotlin/uk/gov/justice/digital/hmpps/prisonersearch/search/services/attributesearch/AttributeSearchIntegrationTest.kt

```diff
@@ -113,34 +79,37 @@ class AttributeSearchIntegrationTest : AbstractSearchDataIntegrationTest() {
       PrisonerBuilder(prisonerNumber = "C1234CC", firstName = "Jimmy", lastName = "Power"),
     )
 
-    webTestClient.attributeSearch(
-      """
-        {
-          "queries": [
-            {
-              "joinType": "OR",
-              "matchers": [ 
-                {
-                  "type": "String",
-                  "attribute": "firstName",
-                  "condition": "IS",
-                  "searchTerm": "John"
-                },
-                {
-                  "type": "String",
-                  "attribute": "lastName",
-                  "condition": "IS",
-                  "searchTerm": "Jones"
-                }
-              ]
-            }
-          ]
-        }
-      """.trimIndent(),
-    )
-      .expectBody()
-      .jsonPath("$.content.length()").isEqualTo(2)
-      .jsonPath("$.content[0].prisonerNumber").isEqualTo("A1234AA")
-      .jsonPath("$.content[1].prisonerNumber").isEqualTo("B1234BB")
+    val request = RequestDsl {
+      query {
+        joinType = OR
+        stringMatcher("firstName" `is` "John")
+        stringMatcher("lastName" `is` "Jones")
+      }
+    }
```

Comment:
Use a DSL to generate requests for testing. This is mainly for brevity as some of the longer requests will be huge in JSON - even this simple test has cut from 32 lines of JSON to 6 lines of DSL.